lib/Serialization/SerializeSIL.cpp

```diff
@@ -1888,6 +1888,25 @@ void SILSerializer::writeSILInstruction(const SILInstruction &SI) {
         UI->hasOperand() ? addValueRef(UI->getOperand()) : ValueID());
     break;
   }
+  case SILInstructionKind::CXXVirtualMethodInst: {
+    // Format: a type, an operand and a SILDeclRef. Use SILOneTypeValuesLayout:
+    // type, Attr, SILDeclRef (DeclID, Kind, uncurryLevel),
+    // and an operand.
+    const CXXVirtualMethodInst *VMI = cast<CXXVirtualMethodInst>(&SI);
```

Comment:
Please use an auto here. You already have CXXVirtualMethodInst on the same line. In general, we use auto * when the type is on the same line.